test/index.js

```diff
@@ -226,7 +226,24 @@ describe('parseValue', function() {
   });
 
   it('returns the raw value if not an array, object or empty string', function() {
-    expect(123).to.eql(123);
+    expect(parseValue(123)).to.eql(123);
+  });
+
+  it('returns the correct value for a string in an object which contains a comma', function() {
+    expect(parseValue({
+      "key": "value with , somewhere",
+    })).to.eql('(key: "value with , somewhere")')
+  });
+  it('can parse nested maps with invalid keys', function() {
```

Comment:
I'm confused why this test shows up in the diff given that it was merged yesterday. Please rebase if you haven't